### ISSUE TITLE
Integration stopped working after updating to 0.8.0

### ISSUE THREAD

**Issue by cartman10** [make-all/tuya-local#24]
When trying to configure the integration a pop-up message indicates that the config flow could not be loaded ...
Initial configuration after update succeeded once.
Device is a BWT FI 45 heat pump.
Error log : 
[home-assistant.log](https://github.com/make-all/tuya-local/files/6729187/home-assistant.log)
2021-06-28 22:27:16 DEBUG (MainThread) [custom_components.tuya_local] Setting up entry for device: 4771835498f4abecc15f
2021-06-28 22:27:16 INFO (MainThread) [custom_components.tuya_local.device] Creating device: 4771835498f4abecc15f
2021-06-28 22:27:16 INFO (MainThread) [custom_components.tuya_local.device] Setting protocol version for Pool heat pump to 3.3.
2021-06-28 22:27:18 ERROR (MainThread) [homeassistant.components.climate] Error while setting up tuya_local platform for climate
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 250, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/tuya_local/climate.py", line 53, in async_setup_entry
    CONF_TYPE: config[CONF_TYPE],
KeyError: 'type'

Full log attached....

Did I forget something ? 
Really appreciate your help!

**Comment by make-all** [comment]
Sorry, it is a bug I overlooked.  I initially put type under options, as in the new config flow it is not part of the first step.  But later I decided to remove it from the configurable options, as type is required to decide which other options are available, so it becomes more complex if type can be changed.

So now if you go into the options to configure them after the integration is already set up, it saves them without the type, and since type was only saved to options, that means it loses it.

I'll release a new version shortly to fix this issue.  Then another one later today or tomorrow to fix another less urgent  issue I found with the options not getting current values as defaults.
